### PR TITLE
Simple typescript support

### PR DESCRIPTION
**Pull request by mihalcea-bogdan-daniel**
Contribution for issue/feat #6

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -165,7 +165,7 @@ <h2>Resize the window to switch between images</h2>
 
 	</div>
 
-	<script src="/scripts.js"></script>
+	<script src="/vanila-js-scss-seed-project/scripts.bundle.js"></script>
```
Comment (CosticaPuntaru) [inline src/index.html]: `vanila-js-scss-seed-project` should not be part of the script src, we should use <base href/> to fix this, this should work out of the box, I will test it and if it dose not work, make it

File: src/js/main.js [hunk 2]
```diff
@@ -2,11 +2,21 @@
 import { printImportWorks } from "./other-file";
 /* You can even import other resources this way */
 import image from '../assets/images/image.jpg';
-
 console.log('main js file works');
 console.log('here is the path for a asset in js', image);
 printImportWorks();
 
+try {
+	const {Calculator} = require("../typescript/main.ts")
```
Comment (CosticaPuntaru) [inline src/js/main.js]: we should not use bad practices in our example, replace with import

File: webpack.config.js [hunk 3]
```diff
@@ -6,14 +6,15 @@ const filepaths = globule.find('src/**/*.html');
 const {BaseHrefWebpackPlugin} = require('base-href-webpack-plugin');
 const openBrowser = require('react-dev-utils/openBrowser');
 const { execSync } = require('child_process');
+const { mainModule } = require("process");
```
Comment (CosticaPuntaru) [inline webpack.config.js]: unused import

File: src/js/main.js (hunk 2 above)
Comment (mihalcea-bogdan-daniel) [reply src/js/main.js]: I made the change, it is using import now. Looking for [this issue now](https://github.com/Cariera-in-IT/vanila-js-scss-seed-project/pull/8#discussion_r840558193)

File: src/index.html (hunk 1 above)
Comment (mihalcea-bogdan-daniel) [reply src/index.html]: I debugged for some hours and I can't find why I need to add  the repository name into the path to scripts.bundle.js in order to work correctly.

Comment (CosticaPuntaru) [reply src/index.html]: there is a trick in the webpack config that made it work before, `   {search: 'src="/scripts.js"', replace: `src="${baseHref}scripts.js"`},` you need to adjust this line in order to accommodate the changes to the file name

Comment (mihalcea-bogdan-daniel) [reply src/index.html]: @CosticaPuntaru I changed the path search and replace, now it is working correctly.

File: webpack.config.js (hunk 3 above)
Comment (mihalcea-bogdan-daniel) [reply webpack.config.js]: @CosticaPuntaru Removed unused import

**Comment by mihalcea-bogdan-daniel** [comment]
#5 added Netlify support and modified README.md to document Netlify deployment.
+ Custom GitHub commit message on npm run github.

**Pull request closed without merging**